<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_Function</class>
 <widget class="QWidget" name="GUI_Function">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Bodyfunctions</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="GUI_FunctionForm" name="widget" native="true"/>
   </item>
   <item>
    <widget class="GUI_FunctionForm" name="widget_2" native="true"/>
   </item>
   <item>
    <widget class="GUI_FunctionForm" name="widget_3" native="true"/>
   </item>
   <item>
    <widget class="GUI_FunctionForm" name="widget_4" native="true"/>
   </item>
   <item>
    <widget class="GUI_FunctionForm" name="widget_5" native="true"/>
   </item>
   <item>
    <widget class="GUI_FunctionForm" name="widget_10" native="true"/>
   </item>
   <item>
    <widget class="GUI_FunctionForm" name="widget_7" native="true"/>
   </item>
   <item>
    <widget class="GUI_FunctionForm" name="widget_9" native="true"/>
   </item>
   <item>
    <widget class="GUI_FunctionForm" name="widget_6" native="true"/>
   </item>
   <item>
    <widget class="GUI_FunctionForm" name="widget_8" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUI_FunctionForm</class>
   <extends>QWidget</extends>
   <header location="global">gui_functionform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
